packages/dx-react-chart-bootstrap4/src/templates/tooltip/overlay.jsx

```diff
@@ -1,9 +1,9 @@
 import * as React from 'react';
 import * as PropTypes from 'prop-types';
-import { Popover, PopoverBody } from 'reactstrap';
+import { Popover } from '../../../../dx-react-bootstrap4/components/popover';
```

Comment:
You can change this path to `../../../../dx-react-bootstrap4/components` because the `index.js` file exists in a `components` filder.